flag.go

```diff
@@ -136,7 +136,10 @@ type StringSlice []string
 
 // Set appends the string value to the list of values
 func (f *StringSlice) Set(value string) error {
-	*f = append(*f, value)
+	for _, s := range strings.Split(value, ",") {
```

Comment:
No strong opinion. If the delimiter was configurable, this wouldn't solve the fact that environment variables and flags are treated differently.